### ISSUE TITLE
Select all objects (i.e. objects on all pages)

### ISSUE THREAD

**Issue by Ignor4mu5** [minvws/nl-kat-coordination#134]
Hi all, 

Perhaps it would be useful to be able to select all objects in order to set an identical clearance level. As of now, it appears to be necessary to open up different pages (in case more than 150 objects exist). If I would be able to select all objects at once, this would have saved me time. 

Interested to know what you think. Have a good day!

Kind regards,
Brandon

**Comment by underdarknl** [comment]
I agree. Ideally it would be good to have an option that just sends the current 'filter options' to the backend, with the request to do action X with all of the matching items.
Instead of sending in each object for each page (possibly triggering a POST count limit), we should just tell Rocky to iterate over all records in the given set, and perform the action itself.
